test/framework/options.py

```diff
@@ -2067,15 +2110,15 @@ def test_try_with_copy(self):
 
         self.mock_stdout(True)
         self.mock_stderr(True)
-        tweaked_ecs_dir = os.path.join(self.test_buildpath, 'my_tweaked_ecs')
-        self.eb_main(args + ['--try-software=foo,1.2.3', '--try-toolchain=gompi,2018a', tweaked_ecs_dir],
-                     verbose=True, raise_error=True)
+        tweaked_ecs_dir = os.path.join(self.test_buildpath)
+        os.chdir(tweaked_ecs_dir)
```

Comment:
Why was this changed? Doesn't this signal a (breaking) change in behavior?

Reply:
Yeah, it does change behaviour but it's not a breaking change. For the case where you were tweaking things, it always copied the tweaked files to a new directory (even if there was only one path in the list, which is the case in the test). This is different behaviour to what happens in the original use case, where if there was only one file, it would do a simple copy to the destination path. I thought it was probably more important to make the behaviour consistent in all cases.

Reply:
I can revert this, but I should probably take the same approach with `--from-pr`, where you don't necessarily know how many ecs you will end up with.